make/ocm.mk

```diff
@@ -0,0 +1,48 @@
+OCM_IMAGE=registry.svc.ci.openshift.org/openshift/release:intly-golang-1.12
+OCM=ocm
+OCM_CLUSTER_NAME=rhmi-$(date +"%y%m%d_%H%M")
+CLUSTER_INFO_FILENAME = "clusterInfo.json"
+
+.PHONY: ocm/version
+ocm/version:
+	@${OCM} version
+
+ocm/login: export OCM_URL := https://api.stage.openshift.com/
+ocm/login: export OCM_TOKEN := ""
+.PHONY: ocm/login
+ocm/login:
+	@${OCM} login --url=$(OCM_URL) --token=$(OCM_TOKEN)
+
+.PHONY: ocm/whoami
+ocm/whoami:
+	@${OCM} whoami
+
+.PHONY: ocm/execute
+ocm/execute:
+	${OCM} ${CMD}
+
+.PHONY: ocm/get/current_account
+ocm/get/current_account:
+	@${OCM} get /api/accounts_mgmt/v1/current_account
+
+.PHONY: ocm/cluster/list
+ocm/cluster/list:
+	@${OCM} cluster list
+
+.PHONY: ocm/cluster/create
+ocm/cluster/create:
+	@$(eval OCM_CLUSTER_ID=$(shell ${OCM} post /api/clusters_mgmt/v1/clusters --body=ocm/cluster.json | jq -r .id ))
+	@echo "Cluster creation started. Cluster id: ${OCM_CLUSTER_ID}"
+	$(call wait_command, $(OCM) get /api/clusters_mgmt/v1/clusters/${OCM_CLUSTER_ID}/status | jq -r .state | grep -q ready, cluster creation, 120m, 300)
```

Comment:
If we are using jq here we need to make sure it's added into the Dockerfile for CI test runs https://github.com/integr8ly/integreatly-operator/blob/master/openshift-ci/Dockerfile.tools